backend/clubs/views.py

```diff
@@ -1198,9 +1199,35 @@ def create(self, request, *args, **kwargs):
             raise DRFValidationError(
                 detail="Approved activities fair events have already been created. "
                 "See above for events to edit, and "
-                f"please email {settings.FROM_EMAIL} if this is en error."
+                "please email contact@pennclubs.com if this is en error."
             )
 
+        if request.data.get("is_recurring", None) is not None:
+            parent_recurring_event = RecurringEvent.objects.create()
+            event_data = request.data.copy()
+            start_time = parse(event_data.pop("start_time"))
+            end_time = parse(event_data.pop("end_time"))
+            offset = event_data.pop("offset")
+            end_date = parse(event_data.pop("end_date"))
+            event_data.pop("is_recurring")
+            club_code = event_data.pop("club")
```

Comment:
This will allow anyone to create an event for any club, which is a security issue. You need to validate that the user can actually do this. To match other endpoints, you may want to read this from the URL, if applicable.